### PR TITLE
5490 Launch Changes

### PR DESCRIPTION
**Pull request by annekainicUSDS**
Getting the 5490 ready for launch. Does 2 things:

1. Moves the 5490 routes to the prod routes for education
2. Makes changes on the process apply page to include links to the 5490

Should be merged once the 5490 is actually ready to deploy.

### CODE REVIEW

File: content/pages/education/apply.md [hunk 1]
```diff
@@ -76,7 +74,7 @@ If you need to make a change (for example, you’re moving to a new school), man
 <div class="usa-alert usa-alert-warning usa-content va-alert" markdown="1">
 	<div class="usa-alert-body">
 
-##### Forms 22-1990N, 22-5490, and 22-5495
+##### Forms 22-1990N and 22-5495
 
 You must apply for education benefits using eBenefits if you're:
 - A dependent whose sponsor is permanently and totally disabled
```
Comment (annekainicUSDS) [inline content/pages/education/apply.md]: Question for @goldenmeanie: Now that the 5490 will be on Vets.gov, how should this language for the eBenefits redirect be amended? I wasn't sure if it was obvious to remove references to dependents here because the 5495 is for these kinds of dependents too, but only if they are managing an existing benefit. It seemed too wordy to say "A dependent whose sponsor is permanently and totally disabled and who is managing an existing benefit". What do you recommend?

**Review by cvalarida — APPROVED** [review]
Once that language is changed, it looks ready to pull the trigger when we're golden for launch!
